packages/cfa-template/template/frontend/package.json

```diff
@@ -31,6 +24,7 @@
     ]
   },
   "dependencies": {
```

Comment:
Do we need all these dependencies?

Reply:
For now yes, but we will be removing as we go.